### PR TITLE
Remove duplicated words in auto completing

### PR DESCRIPTION
**Pull request by sh19910711**
This pull request has the changes for the auto completion.

* Remove duplicated words in auto completing
  - Some methods (like `public_method`) could appear in the list for the auto completion twice
* Remove unused test helpers for template testing
* Do nothing if no words is passsed to the auto completion
  - It caused some exceptions to handle view of the auto completion

Thank you.

### CODE REVIEW

File: lib/web_console/templates/console.js.erb [hunk 1]
```diff
@@ -45,13 +45,23 @@ function CommandStorage() {
...
 function Autocomplete(words, prefix) {
-  this.words = words.concat(); // to clone object
-  this.words.sort(); // by alphabetically order
+  this.words = betterWords(words);
   this.current = -1;
   this.left = 0; // [left, right)
   this.right = words.length;
   this.confirmed = false;
 
+  function betterWords(words) {
```
Comment (gsamokovarov) [inline lib/web_console/templates/console.js.erb]: Can we drop the duplicated names in the server? I think an `uniq!` call in `Context#extract` can do.

Comment (sh19910711) [inline lib/web_console/templates/console.js.erb]: OK 👍

**Comment by sh19910711** [comment]
Updated:

* Use `SortedSet` in the server to drop the duplication names and sort them.

Thanks!

**Comment by gsamokovarov** [comment]
Thanks!

**Pull request merged**